core/src/main/java/com/novoda/noplayer/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -158,7 +156,4 @@ private void showContainer() {
         videoContainer.show();
     }
 
-    public ExoPlayerTwoFacade getInternalExoPlayer() {
```

Comment:
This made me 😢 

We have `Player` and `ExoPlayerTwoImpl` to prevent clients from having to deal with the underlying `ExoPlayer` but this method provides just that.